Actor.cs

```diff
@@ -0,0 +1,85 @@
+internal class Actor : MonoBehaviour
+{
+	private static List<Actor> _actors = new List<Actor>();
```

Comment:
Apart from adding to this in OnEnable this is never used
It can probably be removed

If it can't be removed & we do have a use for it, we should probably define it somewhere other than in the Actor class so that we have proper tracking of actors & their events, some kind of actor-manager?